src/lib/nft-thumbnail.svelte

```diff
@@ -1,21 +1,42 @@
 <script lang="ts">
 	// TODO all of the avail props are logged out with $$props;
 	console.log($$props);
+	export let formatted_date = new Date($$props.created_date).toLocaleDateString('en-US');
```

Comment:
instead we could do 

```
export let created_date; // automagically pulls it from $$props
$: formatted_date = Date(created_date).toLocalDateSTring('en-US');
```

That way if `created_date` ever changes, the UI will automagically react. 

Here's why this works https://svelte.dev/docs#component-format-script-3-$-marks-a-statement-as-reactive

Reply:
I'm happy to merge and update this later :)